hca2scea-backend/output/TEST01/E-HCAD-50.idf.txt

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
The output directory here should not be saved to git. It is a temporary working directory where the artifacts generated during the execution of the tests are generated.
If this file is meant to be the expected sdrf file for the Test01 dataset, it should be moved to `test/golden/Test01/`